ui/platform_window/x11/x11_window_base.cc

```diff
@@ -505,20 +504,7 @@ void X11WindowBase::OnWMStateUpdated() {
                   : ui::PlatformWindowState::PLATFORM_WINDOW_STATE_NORMAL;
```

Comment:
maybe this line can go away, since NORMAL state is set earlier now?

Reply:
done